src/runtime_src/core/pcie/driver/linux/xocl/subdev/feature_rom.c

```diff
@@ -434,7 +434,6 @@ static int feature_rom_probe(struct platform_device *pdev)
 		rom->header.TimeSinceEpoch);
 	xocl_info(&pdev->dev, "FeatureBitMap: %llx", rom->header.FeatureBitMap);
 
-	xocl_subdev_register(pdev, XOCL_SUBDEV_FEATURE_ROM, &rom_ops);
```

Comment:
for one particular subdevice instance, how do we dynamically revoke/unregister its callback to prevent other parts of driver from running its code?

Reply:
It should be done by using xocl_subdev_offline interface and callbacks